dist/jdataview.js

```diff
@@ -1,23 +1,23 @@
-(function (factory) {
-	var global = this;
-
-	if (NODE || typeof exports === 'object') {
-		module.exports = factory(global);
-	} else
-	if (BROWSER) {
-		if (typeof define === 'function' && define.amd) {
-			define([], function () {
-				return factory(global);
-			});
-		}
-		else {
-			global.jDataView = factory(global);
-		}
-	}
-}(function (global) {
-
-'use strict';
-
+(function (factory) {
```

Comment:
Yeah, could be the reason. I'd like to merge, but this isn't very pleasant if it will jump with each commit. Maybe would be better if you could PR source only, and I'll build & publish from my machine. Thanks for your patience!